lib/bricks-evaluate.js

```diff
@@ -49,6 +49,24 @@ function createContext() {
 	extend(context, require('./bricks-net-api'));
 	
 	
+	// Add a `DemoObject` to demonstrate the JSON-POST.
+	context.DemoObject = function () {
+		return {
+			"string": "string",
+			"number": 123.456,
+			"array": [ 1, 2, 3 ],
+			"object": {
+				"key": "value"
+			}
+		};
+	};
+	context.DemoObject.inspect = context.DemoObject.toString = function () {
```

Comment:
The comment comes above to the whole class. These lines work if you evaluate just `DemoObject`, e.g.:
```
KnowSheet> DemoObject
// A cerealizable class for JSON-POST demonstration.
```